app/lib/theme/widgets/DatePickerWidget.tsx

```diff
@@ -157,6 +157,7 @@ DatePickerWidget.defaultProps = {
   // eslint-disable-next-line react/default-props-match-prop-types
   options: {
     isClearable: false,
+    addHorizontalLine: false,
```

Comment:
Looks great, just need to remove this.

Reply:
I merged and completely missed this, sorry I'll open another PR